tasks/configure-jobs.yml

```diff
@@ -5,18 +5,36 @@
     path: "{{ jenkins_home }}/jobs/{{ item }}"
     state: directory
     mode: 0777
-  with_items: jenkins_jobs
+  with_items: "{{ jenkins_jobs }}"
 
 - name: Jobs are present
-  template:
-    src: "{{ jenkins_source_dir_jobs }}/{{ item }}.xml"
+  copy:
```

Comment:
Why copy instead of template?

Template would be better so that the config files can contain ansible variables, no?